department-of-veterans-affairs/va.gov-cms
# VACMS-7072 claro text hierarchy

## Description

Closes #7072. 
- Adds CSS properties & mixins for text styles defined in figma https://www.figma.com/file/ScABI31f3MRFEvQ8WitVPa/Design-System-Draft?node-id=41%3A2131
- Adjusts spacing of various text & containing elements per figma
- Adjusts imported font to use correct weights (600 vs 700), fixing rendering discrepancy between figma & browsers

## Testing done
- Applied text fixes, checked various content types to see that things rendered correctly and spacing didn't feel off.

There are still some iffy areas, particularly in nested paragraphs that allow multiple items (Q&A, accordions, etc) but overall edit forms should feel much more cohesive.

## Screenshots
**before:**

![Screen Shot 2022-01-31 at 4 02 44 PM](https://user-images.githubusercontent.com/11279744/151893242-e481d770-fbd4-43b4-bc81-68ce794851dd.png)
![Screen Shot 2022-01-31 at 4 03 02 PM](https://user-images.githubusercontent.com/11279744/151893245-d8a81f0e-2a89-49c2-a900-ffc63ffbce88.png)
![Screen Shot 2022-01-31 at 4 05 40 PM](https://user-images.githubusercontent.com/11279744/151893246-4c4861f2-0f20-4621-a094-89bf27fddc37.png)


**after:**
![Screen Shot 2022-01-31 at 4 07 57 PM](https://user-images.githubusercontent.com/11279744/151893442-ecc6fe93-8d02-45f9-9b62-df6f912846c6.png)
![Screen Shot 2022-01-31 at 4 08 21 PM](https://user-images.githubusercontent.com/11279744/151893443-1d141b86-238c-4974-85b3-5764123d7303.png)
![Screen Shot 2022-01-31 at 4 08 32 PM](https://user-images.githubusercontent.com/11279744/151893444-265a937f-52c5-47c0-8688-6e95e7c7858b.png)
![Screen Shot 2022-01-31 at 4 08 43 PM](https://user-images.githubusercontent.com/11279744/151893446-a9410d0b-6d78-4a0e-855c-f8a01d8c9394.png)
![Screen Shot 2022-01-31 at 4 05 14 PM](https://user-images.githubusercontent.com/11279744/151893466-33fad361-a22e-45fb-8f52-f0069a1c3a1a.png)


## QA steps

As user, 
check edit forms of various content types and see that the visual hierarchy of text on the page makes more sense.

/node/3776/edit
/node/6359/edit
/node/318/edit

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
